backend/index.ts

```diff
@@ -1,15 +1,19 @@
+import cookieParser from "cookie-parser";
 import express from "express";
 
 import fileRoutes from "./src/routes/fileRoutes";
 // const fileRoutes = require("./src/routes/fileRoutes.ts");
+import loginRoutes from "./src/routes/loginRoutes";
 
 const PORT: string | number = process.env.PORT ?? 3001;
```

Comment:
We want to keep the 3001 inside the env file. 
To do so you'll need to `import dotenv from "dotenv"` and do the corresponding `npm install dotenv`. You'll also need `dotenv.config();` at the start

Note [line 11 ](https://github.com/TritonSE/SDCTA-Data-Visualization-Website/blob/main/server/index.js#L11) from the file. 

Inside the env tile it'll be in the formate of `PORT=whateverportnumber` Similarly for other information its that formate to where instead of PORT it is what ever variable you are calling from process.env."whatever"